projects/netgrif-components-core/src/lib/data-fields/i18n-field/i18n-text-field/abstract-i18n-text-field.component.ts

```diff
@@ -138,12 +138,22 @@ export abstract class AbstractI18nTextFieldComponent extends AbstractI18nErrorsC
             : this.textI18nField.value.defaultValue;
     }
 
+    public getDefaultLanguageCode(): string {
+        return DEFAULT_LANGUAGE_CODE;
+    }
+
     public isPlainText(): boolean {
         if (this.textPropertyEnabled('plainText')) {
             return this.textI18nField.component.properties.plainText === 'true';
         }
     }
 
+    public isBoldText(): boolean {
+        if (this.textPropertyEnabled('boldText')) {
+            return this.textI18nField.component.properties.plainText === 'true';
```

Comment:
this.textI18nField.component.properties.boldText === 'true';

Reply:
Done